scripts/lamp

```diff
@@ -1,7 +1,32 @@
 #!/usr/bin/env bash
 # Because genies come from lamps.
 
+set -eou pipefail
+
+DIR=$(pwd)
+
 # for mongodbtoolchain python3
 export PATH=/opt/mongodbtoolchain/v3/bin:$PATH
 
+if [[ "$*" != *-g* ]]
+then
+    if [[ ! -d venv ]]
+    then
+        echo "Installing virtual environment."
+        echo "To run without a virtual environment, use the -g or --run-global option."
+        python_path=/opt/mongodbtoolchain/v3/bin/python3
+        if [[ ! -e "$python_path" ]]; then
+            python_path="$(command -v python3)"
+        fi
+        echo "creating new env with python: ${python_path}"
+        /usr/bin/env virtualenv -q "${DIR}/venv" --python="${python_path}"
```

Comment:
Couple of comments.
1. I don't think you need `/usr/bin/env` is necessary outside of `#!`, because it just looks at the first thing in your path anyway.
2. Python3's virtualenv is called `venv` and is 90% similar to virtualenv, you'll want to use the following syntax here:
`python3 -m venv venv`
3. related to the above point, you can remove the code related to python_path in this file. Instead, just add `/opt/mongodbtoolchain/v3/bin/python3` to the front of $PATH and call `python3` directly